js/components/time.js

```diff
@@ -78,5 +78,12 @@ Fliplet.FormBuilder.field('time', {
     }
 
     $vm.$v.$reset();
+  },
+  watch: {
+    value: function(val) {
+      if (!val) {
+        this.updateValue(moment(new Date()).format('HH:mm'));
```

Comment:
`moment()` creates a moment object based on the current stamp. `new Date()` is not necessary.

```suggestion
        this.updateValue(moment().format('HH:mm'));
```

Reply:
Fixed.